src/style.css

```diff
@@ -1 +1,59 @@
-/* add styles here */
+body {
+  margin: 0;
```

Comment:
As per the checklist, it's recommended to avoid using tag names for styling (except for `html` and `body`). However, in this case, it's acceptable since you're applying a universal style that should affect the entire body of the document.